src/lib/expansion/expansion-panel.ts

```diff
@@ -93,6 +93,16 @@ export class MatExpansionPanel extends _MatExpansionPanelMixinBase
   /** Whether the toggle indicator should be hidden. */
   @Input() hideToggle: boolean = false;
 
+  /** The positioning of the expansion indicator. */
+  @Input()
+  get togglePosition(): MatAccordionTogglePosition {
+    return this.accordion ? this.accordion.togglePosition : this._togglePosition;
```

Comment:
Shouldn't the more specific position trump the parent?

Reply:
Currently the `hideToggle` value goes based on parent's value even if the child is defined so I went with the same.  There is a strong argument that they both should be changed.